### ISSUE TITLE
CMIP6: Vertical cell thickness

### ISSUE THREAD

**Issue by jbusecke** [m2lines/ocean_emulators#16]
I have so far found the following output for vertical cell thickness: 

1. `thkcello`: From the metadata `thkcello is the nominal cell thickness in z* coordinates. The model actual thkcello is time-dependent and can be calculated as thkcello * ( deptho + zos ) / deptho` **available for all grid labels**
2. `volcello`: This is time dependent ocean cell volume **only available for grid_label='gn'**

So we can get the actual vertical cell extent `dz` by either following instructions in 1. or dividing the volume by area?

@adcroft do you have a feeling if one of these is preferrable? Is the vertical regridding from MOM6 actually converted to z* or to constant z levels?

 If 1. is fine, I think that would be the more consistent way to do it for many models.

**Comment by adcroft** [comment]
Yes, I think 1 is best, especially since it's available for all grids.
